lib/lumberjack/client.rb

```diff
@@ -4,6 +4,7 @@
 require "thread"
 require "openssl"
 require "zlib"
+require "json"
```

Comment:
added class variable Lumberjack::Client.json. To be overwritten by logstash lumberjack input plugin to use LogStash::Json.method(:dump)